lib/katello/engine.rb

```diff
@@ -26,7 +26,12 @@ class Engine < ::Rails::Engine
         :redhat_repository_url => 'https://cdn.redhat.com',
         :consumer_cert_rpm => 'katello-ca-consumer-latest.noarch.rpm',
         :consumer_cert_sh => 'katello-rhsm-consumer',
-        :event_daemon_enabled => true,
+        :event_daemon => {
+          enabled: true,
+          multiprocess: true,
+          lock_file: '/tmp/katello_event_daemon.lock',
+          pid_file: "#{Rails.root.join('tmp')}/pids/katello_event_daemon.pid"
+        },
```

Comment:
In what cases would I as a Foreman administrator want to change each of these settings?

Reply:
I looked closely and in prod we symlink the rails tmp directory to /var/run/foreman. That's great! So I think I can simply hard-code these. My concern was about writable directories between dev and prod